pkg/common/args.go

```diff
@@ -87,7 +89,7 @@ func InitArgs() (*ArgConfig, error) {
 
 	flag.StringVar(&cfg.Focus, "focus", "", "focus runs a specific e2e test. e.g. - sig-auth. allows regular expressions.")
 	flag.StringVar(&cfg.Skip, "skip", "", "skip specific tests. allows regular expressions.")
-	flag.StringVar(&cfg.ConformanceImage, "conformance-image", containerImage,
+	flag.StringVar(&cfg.ConformanceImage, "conformance-image", fmt.Sprintf("registry.k8s.io/conformance:%s", containerImage),
```

Comment:
Let's move the whole image up to the variable definition. I don't think we can guarantee that the image will always be `registry.k8s.io/conformance` and we want to give users the opportunity to override it.